plugins/module_utils/_SqlServerUtils.psm1

```diff
@@ -63,12 +33,12 @@ function Format-JsonOutput {
         # If present, only use default property set
         $defaultPropertySet = $Object.PSStandardMembers.DefaultDisplayPropertySet.ReferencedPropertyNames
 
-        $output = $Object | Select-Object -Property $defaultPropertySet -ExcludeProperty $excludeProperty | ConvertTo-Json -Depth 0 -EnumsAsStrings
+        $output = $Object | Select-Object -Property $defaultPropertySet -ExcludeProperty $excludeProperty | ConvertTo-Json -Depth 1 -EnumsAsStrings
```

Comment:
Just commented on the merged PR but `-NumAsStrings` is not available in Windows PowerShell so this would fail on Windows hosts.

Also may want to make `-Depth` configurable, but with a low default.